app/boards/shields/zmk_uno/zmk_uno.keymap

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2022 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#include <behaviors.dtsi>
+#include <dt-bindings/zmk/keys.h>
+#include <dt-bindings/zmk/backlight.h>
+#include <dt-bindings/zmk/bt.h>
+#include <dt-bindings/zmk/ext_power.h>
+#include <dt-bindings/zmk/outputs.h>
+#include <dt-bindings/zmk/rgb.h>
+
+/* Uncomment if using the "Direct Wire" jumper to switch the matrix to a direct wire.
+
+&kscan_direct_comp { status = "okay"; }
+&kscan_direct { status = "okay"; }
+&kscan_matrix_comp { status = "disabled"; }
+&kscan_matrix { status = "disabled"; }
```

Comment:
These should be commented out, right?

Reply:
Oh. I missed that the comment on line 15 doesn't end. I'd recommend making that be a separate comment so you can just select the block and hit your IDE's uncomment shortcut (Ctrl+K Ctrl+U in VS Code for example).

Reply:
Good call. Updating.